GitUI/UserControls/FileStatusList.cs

```diff
@@ -295,12 +293,22 @@ public IReadOnlyList<GitItemStatus> GitItemStatuses
         {
             get
             {
-                return GitItemStatusesWithDescription?.SelectMany(tuple => tuple.statuses).AsReadOnlyList()
+                return GitItemStatusesWithDescription?.SelectMany(tuple => tuple.Statuses).AsReadOnlyList()
                        ?? Array.Empty<GitItemStatus>();
             }
         }
 
-        private IReadOnlyList<(GitRevision firstRev, GitRevision secondRev, string summary, IReadOnlyList<GitItemStatus> statuses)> GitItemStatusesWithDescription
+        private class FileStatusWithDescription
```

Comment:
Please move to own file. 
Randomly sprinkling the code with additional types increases maintenance overhead.